fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/ExpressionStatisticCalculator.java

```diff
@@ -158,8 +158,10 @@ public ColumnStatistic visitCall(CallOperator call, Void context) {
         private ColumnStatistic nullaryExpressionCalculate(CallOperator callOperator) {
             switch (callOperator.getFnName().toLowerCase()) {
                 case FunctionSet.COUNT:
-                    return new ColumnStatistic(0, inputStatistics.getOutputRowCount(), 0,
-                            callOperator.getType().getTypeSize(), rowCount);
+                    return new ColumnStatistic(0,
+                            Double.isNaN(inputStatistics.getOutputRowCount()) ? Double.POSITIVE_INFINITY :
```

Comment:
How could it be `isNaN` ?

Reply:
@wanpengfei-git As shown in UT, subquery is max(string) because `ColumnStatistic` does not deal with string max/min yet